### DISCUSSION TITLE
Adaptation of the UI for a 7" display

### DISCUSSION THREAD

**Discussion by Paradoxa** [photobooth-app/photobooth-app#162] [Q&A]
Hello everyone,

how can I adapt the buttons for a 7" display?
These are not adapted to the screen and are the same size as on a 22" display.

Thanks in advance

**Comment by mgineer85** [comment]
Changes Not implemented currently.

I use 7“, 10“ and 34“ screens and it’s ok for me on all screens. What would you like to modify?

**Reply by Paradoxa** [reply]
For me the screen is displayed as follows.
My display resolution is 800x480. I can't change this.
Maybe my settings are wrong.

![k20240125_072650](https://github.com/photobooth-app/photobooth-app/assets/19489614/9e6b59db-a575-4596-8bd5-da0b07953619)

**Reply by mgineer85** [reply]
I see. This is a very low resolution.
I will check how of can be handled later.

**Reply by mgineer85** [reply]
Next release will have some minor improvements on the frontpage for low resolution screens:
Text is hidden on small screens and also font-size is a little lower now.
![image](https://github.com/photobooth-app/photobooth-app/assets/3868450/76a22186-a68a-41da-ae75-3ae06f95ffbf)

I added a lowest screen resolution to the prerequisites also to guide new users on choosing a proper screen from the very beginning.

Reference:
https://github.com/photobooth-app/photobooth-app/commit/8ee4d6fa5db2eb9823ff48ad9d5f8356b47e2eb4

**Reply by Paradoxa** [reply]
Thank you.
